data/json/npcs/isherwood_farm/NPC_Barry_Isherwood.json

```diff
@@ -17,7 +17,13 @@
     "id": "NC_ISHERWOOD_BARRY",
     "name": { "str": "Barry Isherwood" },
     "job_description": "Missing member of the Isherwood family.",
-    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "traits": [ 
+      { "group": "NPC_starting_traits" }, 
+      { "group": "Appearance_demographics" },
+      { "trait": "IGNORE_SOUND" },
```

Comment:
If you give Barry  these traits will it break his quest?  Will he return to the migo cell?  Unless you've tested his interactions he's probably not a good fit for these traits.